### PR TITLE
Feature utility props

### PR DESCRIPTION
**Pull request by TayyabDev**

### CODE REVIEW

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

File: dev/com.ibm.ws.install.featureUtility/resources/com/ibm/ws/install/featureUtility/internal/resources/FeatureUtilityToolOptions.nlsprops [hunk 1]
```diff
@@ -83,8 +88,24 @@ installServerFeatures.option-key.--verbose=\
 installServerFeatures.option-desc.--verbose=\
 \tDisplay additional debug information while the action runs.
 #------------------------------\n at 72 chars -- leading tab-----------\n\#
+installServerFeatures.option-key.--acceptLicense=\
+\ \ \ \ --acceptLicense
+installServerFeatures.option-desc.--acceptLicense=\
+\tAccepts the  license agreement.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.install.featureUtility/resources/com/ibm/ws/install/featureUtility/internal/resources/FeatureUtilityToolOptions.nlsprops:94]: Same thing as above comment

File: dev/com.ibm.ws.install.featureUtility/resources/com/ibm/ws/install/featureUtility/internal/resources/FeatureUtilityMessages.nlsprops [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+MSG_NO_CONFIG_PROXY=No proxy configured
+MSG_REPO_FILE_NOT_FOUND=No user configuration detected. The IBM WebSphere Liberty Repository is the default assets repository.
+MSG_REPO_FILE_NOT_FOUND_EXPLANATION=To customize installUtility settings, create a repositories.properties file at the following location: {0}
+MSG_REPO_FILE_NOT_FOUND_ACTION=Use the following sample as a template to create your own repositories.properties file. Uncomment the lines marked by a single # character, and replace the values with your own customized values.
+MSG_VALIDATION_SUCCESSFUL=The properties file successfully passed the validation.
+MSG_VALIDATION_FAILED_NO_MESSAGES=The properties file failed the validation. Use the --viewValidationMessages option to view detailed validation messages.
+MSG_VALIDATION_NUM_OF_ERRORS=Number of errors: {0}
+MSG_VALIDATION_MESSAGES=Properties File Validation
+MSG_VALIDATION_MESSAGE_FORMAT={0}. {1}
+MSG_PASSWORD_NOT_ENCODED=The password for this repository is not encoded. You can encode passwords by running the securityUtility encode command with the --encoding option set to a supported encoding type, which are xor (default), aes, and hash.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.install.featureUtility/resources/com/ibm/ws/install/featureUtility/internal/resources/FeatureUtilityMessages.nlsprops:38]: Suggested update:
"The password for this repository is not encoded. You can encode passwords by running the securityUtility encode **action** with the --encoding option set to a supported encoding **type. The supported types are xor** (default), aes, and hash."

**Comment by TayyabDev** [comment]
Fixed up the messages according to your feedback. Thanks!

**Comment by TayyabDev** [comment]
#build

**Comment by TayyabDev** [comment]
#build

**Pull request closed without merging**